projects/components/src/select/select.component.ts

```diff
@@ -61,7 +64,7 @@ import { SelectSize } from './select-size';
             <div
               *ngSwitchCase="'${SelectTriggerDisplayMode.Icon}'"
               class="trigger-content icon-only"
-              [ngClass]="this.selected !== undefined ? 'selected' : ''"
+              [ngClass]="[this.selected !== undefined ? 'selected' : '', this.popoverOpen ? 'open' : '']"
```

Comment:
may be we should make this an object for better readability

`"{ selected: this.selected !== undefined, open: this.popoverOpen}"`

Reply:
done